qpc_tools/cli/install.py

```diff
@@ -69,6 +69,7 @@ def _validate_args(self):
                 print(_(messages.CLI_INSTALL_MUST_SPECIFY_PORT_AND_HOST))
                 sys.exit(1)
             self.args.configure_server = 'true'
+        validate_and_update_paths(self.args)
```

Comment:
I am not a huge fan of this name, it seems to general. What do you think about `require_absolute_paths`

Reply:
🌮

Reply:
Changed to what we discussed